src/shared/services/fields/fields.utils.ts

```diff
@@ -15,7 +15,8 @@ export class FieldsUtils {
 
   private static readonly currencyPipe: CurrencyPipe = new CurrencyPipe('en-GB');
   private static readonly datePipe: DatePipe = new DatePipe(new FormatTranslatorService());
-  public static readonly LABEL_SUFFIX = '-LABEL';
```

Comment:
@paul-graham do you think it's possible any Services might have tests that depend on label fields ending in `-LABEL`? Might be something we need to communicate to them, in case it breaks their tests - a bit like what happened with the change in some element IDs.

Reply:
This is held entirely internally so there shouldn't be an issue with this - there are no controls appearing anywhere on the screen with a `-LABEL` (or now `---LABEL`) suffix. It's simply a lookup for `MultiSelect` fields.

Reply:
Ah, OK - great. Thanks for explaining, @paul-graham.